### PR TITLE
PNDA-1960 - Make Kafkat available on nodes as option for Kafka management cli

### PR DESCRIPTION
**Pull request by boopalans**
Adding kafkat as one of the upstream build.

### CODE REVIEW

File: mirror/dependencies/pnda-rpm-package-dependencies.txt [hunk 1]
```diff
@@ -30,13 +30,15 @@ MySQL-python-1.2.5-1.el7
 nginx
 nmap-ncat
 ntp-4.2.6p5
+patch
```
Comment (jgarnier) [inline mirror/dependencies/pnda-rpm-package-dependencies.txt]: patch need to be part of deb dependencies

File: mirror/dependencies/pnda-rpm-package-dependencies.txt [hunk 2]
```diff
@@ -30,13 +30,15 @@ MySQL-python-1.2.5-1.el7
...
 python2-pip
 python34-devel
 python34-pip
 python-carbon-0.9.15-6.el7
 python-devel
 redhat-lsb
 redis-3.2.3-1.el7
+ruby-devel
```
Comment (jgarnier) [inline mirror/dependencies/pnda-rpm-package-dependencies.txt]: ruby-devel need to be part of deb dependencies

File: mirror/dependencies/pnda-rpm-package-dependencies.txt (hunk 1 above)
Comment (boopalans) [reply mirror/dependencies/pnda-rpm-package-dependencies.txt]: Added

File: mirror/dependencies/pnda-rpm-package-dependencies.txt (hunk 2 above)
Comment (boopalans) [reply mirror/dependencies/pnda-rpm-package-dependencies.txt]: Added

File: build/upstream-builds/build-kafkatool.sh [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+echo -n "shyaml: "
+if [[ -z $(which shyaml) ]]; then
+    error
+else
+    echo "OK"
+fi
+
+if [[ ${MODE} == "PNDA" ]]; then
+    KT_VERSION=$(wget -qO- https://raw.githubusercontent.com/boopalans/platform-salt/${ARG}/pillar/services.sls | shyaml get-value kafkatool.release_version)
```
Comment (jgarnier) [inline build/upstream-builds/build-kafkatool.sh]: need to be pndaproject repo

File: build/upstream-builds/build-kafkamanager.sh [hunk 4]
```diff
@@ -53,3 +53,13 @@ sbt clean dist
 cd ..
 mv kafka-manager-${KM_VERSION}/target/universal/kafka-manager-${KM_VERSION}.zip pnda-build/
 sha512sum pnda-build/kafka-manager-${KM_VERSION}.zip > pnda-build/kafka-manager-${KM_VERSION}.zip.sha512.txt
+
+#kafkat build process
+git clone https://github.com/airbnb/kafkat.git
```
Comment (jgarnier) [inline build/upstream-builds/build-kafkamanager.sh]: need to be removed as kafkat has its own upstream build script

**Pull request merged**